deploy/kessel-inventory.yaml

```diff
@@ -18,7 +18,11 @@ objects:
           psk:
             pre-shared-key-file: /psks/psks.yaml
         authz:
-          impl: allow-all
+          impl: kessel
+          kessel:
+            insecure-client: true
+            url: kessel-relations-api:9000
+            enable_oidc_auth: false
```

Comment:
IMO we should try to user clowder for these - as different environments might have different setups on how to talk with the different services. Usually requiring a more secure protocol or trying something new.

That said, I think this is perfectly fine to start with!